webapp/src/main/java/com/box/l10n/mojito/okapi/filters/XMLEncoder.java

```diff
@@ -49,9 +56,17 @@ public String escape(String text) {
             text = escapeSingleQuotes(text);
         }
         return enclosedInDoubleQuotes ? "\"" + text + "\"" : text;
-        
+
     }
-    
+
```

Comment:
Can you document the reasoning behind this function because it is not obvious and at least I will forget quickly :)